public/javascripts/items.js

```diff
@@ -79,21 +77,54 @@ const completedClicked = event => {
     .then( result => result.json() )
     .then( checkJsonForSuccessField )
     .then( json => {
-      const parent = $( `.title[data-id=${id}]` )
+      const parent = $( `.item__title[data-id=${id}]` )
         element.data( 'completed', completed )
         if( completed ) {
-          parent.addClass( 'completed' )
+          parent.addClass( 'item__title--completed' )
         } else {
-          parent.removeClass( 'completed' )
+          parent.removeClass( 'item__title--completed' )
         }
       }
     )
   }
 
+  const getFilterStatus = () => {
+    const pageURL= decodeURIComponent(window.location.search)
+    const parameterMatch = /completed_filter=(.*)/
+    let filterStatus = parameterMatch.exec(pageURL)
+    if(filterStatus === null){
```

Comment:
I think we should be able to remove this if statement, since the `default` case in the switch statement would presumably be invoked when the filterStatus does not match. We can also change filterStatus to a value once we pass the request object in (actually, passing in the value would probably be more appropriate than passing the entire request object), and greatly simplify this function.